src/main/scala/chiseltest/stage/ChiselTestOptions.scala

```diff
@@ -2,49 +2,61 @@ package chiseltest.stage
 
 import chisel3.{Data, MultiIOModule}
 import chiseltest.ChiselTestException
-import chiseltest.backends.SimulatorInterface
+import chiseltest.backends.{SimulatorBackend, SimulatorInterface}
 import firrtl.ir.Port
 
 class ChiselTestOptions private[stage] (
```

Comment:
This class is really looking like the [God object](https://en.wikipedia.org/wiki/God_object) antipattern, especially because it now has a bunch of options that don't make sense in all backends.

Reply:
It's used for `firrtl.options.Viewer`, convert a `AnnotationSeq` to required data.